src/components/collection/UnpublishedItemAlert.tsx

```diff
@@ -15,7 +15,7 @@ type Props = {
   canRead: boolean;
   canPublish: boolean;
   isPublished: boolean;
-  currentMember?: CompleteMember | null;
+  currentMember?: CompleteMember | CompleteGuest | null;
```

Comment:
```suggestion
  currentMember?: CurrentAccount;
```

Reply:
Nice thanks!

Reply:
But as discussed with @spaenleh, I should return null if it is not an real account.